.github/workflows/rhel-9-4.yml

```diff
@@ -54,1236 +53,440 @@ jobs:
-          state: pending
-          description: 'Runner has been deploying...'
-          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-  edge-commit-94:
-    needs: [pr-info, pre-edge-commit-94]
-    runs-on: [kite, x86_64, rhos-01, rhel-9-4, large]
-    env:
-      STATUS_NAME: edge-commit-9.4
-
-    steps:
-      - name: Create in-progress status
-        uses: octokit/request-action@v2.x
-        with:
-          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
-          context: ${{ env.STATUS_NAME }}
-          state: pending
-          description: 'Test has been running...'
-          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Clone repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  # v4
         with:
           ref: ${{ needs.pr-info.outputs.sha }}
           fetch-depth: 0
 
-      - name: run ostree.sh
-        run: ./ostree.sh
-        env:
-          DOWNLOAD_NODE: ${{ secrets.DOWNLOAD_NODE }}
-        timeout-minutes: 60
-
-      - name: Set non cancelled result status
-        if: ${{ !cancelled() }}
-        uses: octokit/request-action@v2.x
-        with:
-          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
-          context: ${{ env.STATUS_NAME }}
-          state: ${{ job.status }}
-          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Set cancelled result status
-        if: ${{ cancelled() }}
-        uses: octokit/request-action@v2.x
-        with:
-          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
-          context: ${{ env.STATUS_NAME }}
-          state: error
-          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - uses: actions/upload-artifact@v3
-        if: ${{ always() }}
-        with:
-          name: edge-commit-9.4
-          path: |
-            *.json
-            *.log
-
-  pre-edge-installer-94:
+      - name: Run the tests
+        uses: sclorg/testing-farm-as-github-action@v2
+        with:
+          compose: RHEL-9.4.0-Nightly
+          arch: x86_64
+          api_key: ${{ secrets.TF_API_KEY }}
+          git_url: ${{ needs.pr-info.outputs.repo_url }}
+          git_ref: ${{ needs.pr-info.outputs.ref }}
+          update_pull_request_status: true
+          pull_request_status_name: "edge-commit-9.4"
+          tf_scope: private
+          tmt_plan_regex: edge-commit
```

Comment:
The tmt plan regex should be `edge-test` and use only one to run all palns in edge-test.fmf. We can't split the tests because each separated test will consume one github runner. The whole org only has 20 concurrent runner limitation.

Reply:
fixed